client/client.go

```diff
@@ -190,7 +193,9 @@ func Transfer(agent, src, dst string) error {
 		select {
 		case r := <-out:
 			if r.Error != nil {
-				log.Println("ERROR fail with transfer request to", r.Url)
+				log.WithFields(log.Fields{
+    			"Url": r.Url,
+  			}).Error("ERROR fail with transfer request to", r.Url)
```

Comment:
Here we can take advantage of logrus to report Error so we don't need ERROR in error message.